### PR TITLE
change writing-documentation.rst to write-documentation.rst

### PR DESCRIPTION
**Pull request by JDRanpariya**
#### What does this implement/fix? Explain your changes.

Its first contribution. As described in getting started with contribution flower is following Diàtaxis framework. This PR changes the documentation from writing-documentation.rst to write-documentation.rst

### CODE REVIEW

File: doc/source/write-documentation.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Writing documentation
+How to write documentation
 =====================
```
Comment (danieljanes) [inline doc/source/write-documentation.rst:2]: The `===` on this line should be as long as the line above

**Review by danieljanes — CHANGES_REQUESTED** [review]
Thanks for the PR @JDRanpariya, looks good! I left one tiny comment, once that is resolved we can merge it 🚀

File: doc/source/write-documentation.rst (hunk 1 above)
Comment (JDRanpariya) [reply doc/source/write-documentation.rst:2]: Oh I get it now. It was mentioned in the doc but didn't quite get it.I will make required changed and push it.

**Review by danieljanes — APPROVED** [review]
Lgtm, thanks for the contribution!

**Pull request merged**